src/our-work/index.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: layouts/default.njk
+title: Our Work
+---
+
+## Partners
+
+### Piikup
```

Comment:
I like the idea of referencing Piikup, but we should check in with April before we push it to production, just to be sure she's ok with that.

Reply:
Absolutely. We can omit for now and then work on a case study with her separately if she's interested.